kpet/run.py

```diff
@@ -127,7 +113,8 @@ def __get_hosts(database, src_path_set):
                         case.host_type_regex or \
                         suite.host_type_regex or \
                         database.host_type_regex
```

Comment:
I can't follow all the logic of the code, but I just need to ask: what's the difference between database.host_type_regex and host_type_regex?

Reply:
`database.host_type_regex` is a host type regex assigned at the DB-level (global default), in the database. The `host_type_regex` is the host type regex in effect for the particular test case we're iterating through, which takes into account host type regex'es at the database level (`database.host_type_regex`), suite level (`suite.host_type_regex`) and case level (`case.host_type_regex`), which override each other in this order, if defined.